10-flow-rate-monitor/README.md

```diff
@@ -0,0 +1,113 @@
+
+With this your Notehub backend is fully configured, and you’re ready to start
+setting up your hardware.
+
+## Hardware
+
+The following hardware is required to run the Flow Rate Monitor project.
+
+* A [Blues Starter Kit](https://shop.blues.io/collections/blues-starter-kits).
+* A flow meter, such as this [GREDIA Quick Connect Water Flow Sensor](https://www.amazon.com/dp/B07RD4JXLY/ref=cm_sw_r_api_i_652XGBZAS7RN06BSSHRT_0).
+* Tubing for moving liquid throughout your hardware, such as this
+[1/4 OD silicone tubing](https://www.amazon.com/dp/B09H4RNGGG/ref=cm_sw_r_api_i_H171CC4D2EDPPM23X8A4_0?th=1).
+* A power breakout, such as this [HiLetgo power breakout](https://www.amazon.com/dp/B07X9SQKL6/ref=cm_sw_r_api_i_5JYD92FH343E04VKPMR5_0)
+for supplying correct voltage to this project’s various components.
+* An I2C level shifter, such as [this one from HiLetgo](https://www.amazon.com/dp/B07F7W91LC/ref=cm_sw_r_api_i_FZPJ7VRY2329ARNQ3W22_0), to
+translate the flow meter’s pulses from 5V logic to 3.3V logic for the Swan
+microcontroller.
+
+Additionally you may wish to also use the following:
+
+* (Optional) An enclosure for your hardware such as this
+[outdoor-friendly enclosure from Sixfab](https://sixfab.com/product/raspberry-pi-ip54-outdoor-iot-project-enclosure/).
+
+Once you have all of your hardware you’ll next need to assemble the pieces.
+To start, complete the [Notecard and Notecarrier-F quickstart guide](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/), which will help
+you connect your Notecard, Notecarrier, and Swan.
+
+![The Notecard, Notecarrier, and Swan connected](images/notecard-and-notecarrier.jpg)
+
+From there you’ll need to connect your flow meter to the Notecarrier. If you’re using this project’s recommended hardware, here’s a [wiring diagram that shows how all the components connect](https://miro.com/app/board/uXjVP1OBst0=/).
+
+![Wiring diagram of the various flow rate monitor components](images/wiring-diagram.png)
+
+Regardless of which hardware you use, you’ll need to have the following pin
+wired to your Notecarrier:
+
+* The flow meter’s signal line must be be connected to the Notecarrier’s `F_D10`
+pin. The Swan uses this pin to read the flow rate from the flow meter.
+
+When everything is put together your can then place your hardware inside the optional case like the one we recommended from Sixfab, and you should be good to go.
+
+Your next step is getting the project’s firmware running on
+your device.
+
+## Firmware
+
+The Flow Rate Monitor project has custom firmware that runs on the Swan. As a next step,
+complete the [firmware’s setup guide](firmware/) to get your firmware up and running.
+
+## Web Application
+
+The Flow Rate Monitor project’s web application allows you to view flow rates, set alarm thresholds, and manage environment variables in a
+browser.
+
+![The Valve Monitor project’s web application](images/web-app.png)
```

Comment:
This is very trivial, but it’d be nice if the image of the web app showed a number for a flow rate instead of a “-”, just to make this feel like it’s showing a meter actually in use.

Reply:
Can do